### ISSUE TITLE
Add vertical hue slider

### ISSUE THREAD

**Issue by fd-gbouillon** [super-effective/react-color-picker#6]
Can you add an option `vertical || horizontal` to change Hue picker position on the left of Saturation picker.

Something like this :
```js
var updateHue = useCallback(function (evt) {
    var huePosition = getPagePosition(hueSliderRef.current);
    var position = vertical ? evt.pageY - huePosition.top : evt.pageX - huePosition.left;
    var clientSize = vertical ? hueSliderRef.current.clientHeight : hueSliderRef.current.clientWidth;
    var updatedHue = getHueFromPosition(position, clientSize);
    setColorFromHsv(_objectSpread2(_objectSpread2({}, hsvRef.current), {}, {
      hue: updatedHue
}));
```

**Comment by AtomicJon** [comment]
I am considering adding other layouts but first need to decide which to support when I revisit styling the picker. For now, I have published version 1.2.0 which exposes the Saturation/Value selector and the Hue slider (with a vertical option) so that they can be positioned and styled as needed. For example, you can now do the following in your app to layout the color picker with the hue slider vertically on the left:

```
import React, { useState } from 'react';
import { HueSlider, SaturationValueSelector } from '@super-effective/react-color-picker';
import { hsvToHex } from '@super-effective/colorutils';

import styles from './App.module.scss';

const App = () => {
  const [hue, setHue] = useState(171.03896103896102);
  const [saturation, setSaturation] = useState(0.7196261682242991);
  const [value, setValue] = useState(0.8392156862745098);

  const hex = hsvToHex(hue, saturation, value);

  return (
    <div className={styles.app}>
      <h1>{`Color: ${hex}`}</h1>
      <dvi className={styles.color_picker}>
        <div className={styles.hue_slider}>
          <HueSlider
            hue={hue}
            layout={HueSlider.LAYOUTS.VERTICAL}
            onChange={(updatedHue) => setHue(updatedHue)}
          />
        </div>
        <div className={styles.saturation_value}>
          <SaturationValueSelector
            hue={hue}
            saturation={saturation}
            value={value}
            onChange={({ saturation: updatedSaturation, value: updatedValue }) => {
              setSaturation(updatedSaturation);
              setValue(updatedValue);
            }}
          />
        </div>
      </dvi>
    </div>
  );
};

export default App;
```

App.module.scss:
```
.app {
  .color_picker {
    display: flex;
    width: 400px;
    height: 400px;
    padding: 15px 5px;

    border: 1px solid #ccc;
    border-radius: 10px;

    .hue_slider {
      height: 100%;
    }

    .saturation_value {
      flex: 1 1 auto;
    }
  }
}
```

**Comment by AtomicJon** [comment]
Closing as exposed components allow for user defined positioning of controls for now. Will revisit layout options when styling is updated next.
